### DISCUSSION TITLE
Run the script in the appropriate PowerShell version error

### DISCUSSION THREAD

**Discussion by lmfodor** [farag2/Sophia-Script-for-Windows#270] [Q&A] [answered]
Hi Team.
I just downloaded the last version of Sophia Script 6.05 for Windows 11 and the last Wrapper 2.5. When I try to run the script, it gave me this error in the powershell screen

Script     6.0.5      Sophia                              {ActiveHours, AdminApprovalMode, AdvertisingID, AeroShakin...
WARNING: You're trying to run script via PowerShell 5.1. Run the script in the appropriate PowerShell version

What did I do wrong? Thanks for your help!
Best

**Comment by farag2** [comment]
May be you downloaded the PowerShell 7 version? Any screens?

**Comment by lmfodor** [comment]
Yes, I just installed Windows 11. I thought it comes with Powershell 7.. Should I install PS7 or run other Sophia version?

![Powershell version](https://user-images.githubusercontent.com/83538149/136673427-189881af-b0a4-4795-8cb0-35b2f3a7db2e.jpg)

Thanks!

**Comment by farag2** [comment]
Impossible. I just by myself ran it on my PC having downloaded from the GH release page.
Please, send the exact link to the zip archie you downloaded.

**Comment by farag2** [comment]
Ah, so you downloaded the PowerShell 7 version? No, 11 doesn't supports it by default. You need to download `Sophia.Script.for.Windows.11.v6.0.5.zip`

**Comment by lmfodor** [comment]
I installed PowerShell 7.1.4 but when I click "Run PowerShell" in the Wrapper app, it still open the old 5.x version...

**Comment by farag2** [comment]
Hm... It shouldn't. @BenchTweakGaming

**Comment by farag2** [comment, marked as answer]
Stop. If you use PS 7 you need to download the PS 7 module version. `Sophia.Script.for.Windows.11.v6.0.5.PowerShell.7.zip`

**Reply by slebreton79** [reply]
Thats the one I downloaded too  (Sophia.Script.for.Windows.11.v6.0.5.PowerShell.7.zip) and doesn't work  I get same version error

**Reply by farag2** [reply]
And now? This is very strange. Can you provide a new screen?

**Comment by lmfodor** [comment]
Thanks to everyone for your help! I finally ran it with powershell 5.x and it was fine.

**Comment by slebreton79** [comment]
It's working now
